examples/CMakeLists.txt

```diff
@@ -13,4 +13,5 @@ find_package(Boost COMPONENTS program_options)
 if(Boost_FOUND)
   add_subdirectory(viz)
   add_subdirectory(dbscan)
+  add_subdirectory(raytracing)
```

Comment:
If there is no specific rule, I would keep it as raytracing, just like callback and dbscan.
The underscore can be saved for later when the example pool gets really big, so that you can do sth like dbscan_type1_variation3.